pkg/replication/replication.go

```diff
@@ -425,9 +427,17 @@ func (r *replication) SetManifest(man manifest.Manifest) {
 	r.manifest = man
 }
 
+func (r *replication) SetTimeout(timeout time.Duration) {
+	// not changing the manifest technically but this will work, don't expect
+	// timeout changes to be highly concurrent with manifest changes
+	r.mu.Lock()
+	r.timeout = timeout
```

Comment:
I'm reading the code in Enact() and I believe that this timeout will be re-read from the *replication each time the loop ticks. Does that match your assessment?

Reply:
yeah, each time a node is pulled off the queue